src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -11,13 +11,16 @@
 import org.junit.jupiter.api.Test;
 
 class RegistrationServiceImplTest {
-    private static RegistrationServiceImpl REGISTRATION_SERVICE;
-    private static User VALID_USER;
+    private static final int LOGIN_MIN_LENGTH = 6;
+    private static final int PASSWORD_MIN_LENGTH = 6;
+    private static final int MIN_AGE = 18;
+    private static RegistrationServiceImpl registrationServiceImpl;
```

Comment:
its better to declare your service with interface type rather with its implementation class type. For example:

Bad practice:
```java
ArrayList<Integer> listOfIntegers = new ArrayList<>();
``` 

Good practice:
```java
List<Integer> listOfIntegers = new ArrayList<>();
```